### PR TITLE
feat: add project set-up template

### PR DESCRIPTION
**Pull request by Samandar587**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+LOG_LEVEL=info
+RPC_PORT=:5005
+
+POSTGRES_HOST=db
+POSTGRES_PORT=5432
+POSTGRES_DATABASE=marketplace_db
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_SSLMODE=disable
+PGDATA=/var/lib/postgresql/post-datas
```
Comment (khudayberdiyev25) [inline .env.example:13]: you need to add empty line in order to get rid of this red warning.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+*.DS_Store
+.vscode
+.idea
+.idea/
+__MACOSX/
+bin/
+/postgres-data
+.werf_secret_key
+.go-cache
+.go
```
Comment (khudayberdiyev25) [inline .gitignore:12]: need empty line

File: migrations/000001_foobar.down.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+drop table if exists movies;
```
Comment (khudayberdiyev25) [inline migrations/000001_foobar.down.sql:1]: lets remove this dummy migrations.

**Review by khudayberdiyev25 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (Samandar587) [reply .gitignore]: added!

File: migrations/000001_foobar.down.sql (hunk 3 above)
Comment (Samandar587) [reply migrations/000001_foobar.down.sql]: removed

**Review by khudayberdiyev25 — APPROVED** [review]

**Pull request merged**